skfusion/fusion/decomposition/dfmc.py

```diff
@@ -95,7 +94,7 @@ def fuse(self, fusion_graph):
 
         parallelizer = Parallel(n_jobs=self.n_jobs, max_nbytes=1e3, verbose=self.verbose)
         task_iter = (delayed(parallel_dfmc_wrapper)(
-            R=R, M=M, Theta=T, obj_types=object_types,
+            R=R, M=M, Theta=T, obj_types=self.fusion_graph.object_types,
```

Comment:
[Iteration order of obj_types](https://github.com/mims-harvard/scikit-fusion/blob/master/skfusion/fusion/decomposition/_init.py#L13) in matrix initialization is not conserved if `obj_types` is a `set`. Consequently, matrix initialization is not always reproducible even if the same `random_state` is used.

Uniqueness of keys in `self.fusion_graph.object_types` is guaranteed already by [its type](https://github.com/mims-harvard/scikit-fusion/blob/master/skfusion/fusion/base/fusion_graph.py#L32), so conversion into a `set` is redundant.